docs/bin/plugin_formatter.py

```diff
@@ -131,6 +132,16 @@ def rst_xline(width, char="="):
 
     return char * width
 
+def to_boolean(default):
```

Comment:
default in this context does not make any sense to me. could you please pick some name, which would be better reflect your intensions?